app/views/manifests/show.json.rabl

```diff
@@ -0,0 +1,9 @@
+object false
+
+node(:name)        { Setting.site_title }
+node(:description) { strip_tags(Setting.site_description.presence || I18n.t('about.about_mastodon')) }
+node(:icons)       { [{ src: '/android-chrome-192x192.png', sizes: '192x192', type: 'image/png' }] }
```

Comment:
Is it possible to add a 512x512 icon as well? This is nice because it's shown on the "splash screen" when the app is launched from the home screen. If not, no biggie.